examples/continuous-delivery/argocd/kube-prometheus/argocd-basic.jsonnet

```diff
@@ -0,0 +1,13 @@
+local kp = (import 'kube-prometheus/kube-prometheus.libsonnet') +
+           (import 'kube-prometheus/kube-prometheus-eks.libsonnet') + {
```

Comment:
as a basic example I would prefer to skip this eks patch

Reply:
Done.